### DISCUSSION TITLE
I cant run configure.py

### DISCUSSION THREAD

**Discussion by Quack2Play** [riidefi/mkw#276] [General]
When i try to run configure.py i get the message: Traceback (most recent call last):
  File "g:/MKW Code/mkw-master/mkw-master/configure.py", line 11, in <module>
    from random import randbytes
ImportError: cannot import name 'randbytes' from 'random' (C:\Python38\lib\random.py)

**Comment by em-eight** [comment]
randbytes was added in python 3.9, so make sure you use a newer python version

**Comment by Quack2Play** [comment]
i installed a new python and runned the program but then it says: File "g:\MKW\mkw-master\mkw-master\configure.py", line 126, in patch_compilers
    with open(CWCC_PATHS["4201_127"], "rb") as og:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '.\\tools\\4201_142\\mwcceppc.exe' but i have the .exe in the tools folder
